### PR TITLE
Improve unknown feature diagnostic messages

### PR DESCRIPTION
**Pull request by mdlnr**
Solves #38

Message now includes current object EClass.

Example:
```
Unknown feature 'some_unknown_feature' for http://www.eclipse.org/emf/2002/Ecore#//EClass
```

### CODE REVIEW

**Review by vhemery — CHANGES_REQUESTED** [review]
LGTM. Thank you for your contribution.
Though, practice on the project is to update the copyright date-range on edition.
Can you please update "Copyright (c) 2019-2021" to "Copyright (c) 2019-2022" before I merge ?
(you can amend the commit and force push to update the PR)

**Comment by mdlnr** [comment]
Thanks for the feedback. Should look good now.

**Review by vhemery — APPROVED** [review]

**Comment by vhemery** [comment]
Thank you very much @mdlnr for this feature.
As I am quite new among committers, I am just checking with the rest of the team that I can directly process to the merge and that it doesn't conflict with the project's agenda.
I or someone else in the team should merge this very soon.

**Pull request merged**
